### PR TITLE
Update LED Capabilities

### PR DESCRIPTION
**Pull request by yash-garg**
Along with remapping hardware keys

### CODE REVIEW

File: keylayout/ft5x06_ts.kl [hunk 1]
```diff
@@ -25,12 +25,7 @@
 # OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN
 # IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Hardware keys
```
Comment (Shr3ps) [inline keylayout/ft5x06_ts.kl]: Why dropping hardware keys?

Comment (yash-garg) [reply keylayout/ft5x06_ts.kl]: Not dropping ! remapped n removed the comments
Check that commit

Comment (Shr3ps) [reply keylayout/ft5x06_ts.kl]: You removed POWER, VOLUME_UP/DOWN

Comment (yash-garg) [reply keylayout/ft5x06_ts.kl]: Done.

Comment (Shr3ps) [reply keylayout/ft5x06_ts.kl]: Please squash/amend your commits in a proper way.

Comment (yash-garg) [reply keylayout/ft5x06_ts.kl]: okay

**Pull request closed without merging**